livestream/live-2019-09-20.m3u8

```diff
@@ -0,0 +1,529 @@
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:40
+#EXT-X-MEDIA-SEQUENCE:0
+#EXT-X-DISCONTINUITY
+#EXTINF:40.000,
+https://ipfs-gateway.live.mesh.world/ipfs/QmSjEs8kC8ZgLetzfpP7QTY45GVvzGeQRfLh4qkGxcswHT
```

Comment:
@DarkDrgn2k I think we want to take these from ipfs.infura.io once we have them pinned right? If we are going to stream from here, we can probably keep them as relative?

Reply:
new code seems to correct the issue that would arise from this

Tested using `sed "s|https://ipfs-gateway.live.mesh.world||" test1.m3u8` and it seemed to work fine 

However it re-append an ipfs server unless you clear out the value in
 [ipfs_gateway](https://github.com/tomeshnet/ipfs-live-streaming/blob/master/terraform/shared/video-player/js/common.js#L2)

Reply:
How do you think I should get these on ipfs.infura.io / ipfs.io? Now there are 1449 hashes and I am doing:

```
$ grep 'ipfs/' live-*.m3u8 | sed 's|.*ipfs/||g' > ipfs-cids.txt
$ cat ipfs-cids.txt | xargs -I {} curl "https://ipfs.infura.io:5001/api/v0/pin/add?arg=/ipfs/{}&recursive=true&progress=true"
```

Which is taking forever and I don't think infura is actually able to locate some of the chunks... they are all pinned on live.mesh.world right?

Reply:
@darkdrgn2k the `ipfs_gateway` param is a little different in the Our Networks fork, so this is working now :D